spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -49,6 +51,7 @@
       end
 
       before do
+        travel_to Time.utc(2022, 2, 8, 16, 52, 15)
         post "/logs", headers:, params: params.to_json
       end
```

Comment:
I think that you might be able to change this to use Timecop like we do in other time related tests. 

It would look something like this:

```ruby
before do
  Timecop.freeze(Time.utc(2022, 2, 8, 16, 52, 15))
  post "/logs", headers:, params: params.to_json
end

after do
  Timecop.unfreeze
end
```

Reply:
I thought I'd already searched to see if we used Timecop in the codebase but clearly hadn't 🤦‍♂️
Thanks for catching!